### ISSUE TITLE
Update to hash

### ISSUE THREAD

**Issue by naveensrinivasan** [ossf/scorecard-action#63]
https://github.com/ossf/scorecard-action/blob/25e5ca0a67911cbe0bdc55d008768f68716c7df5/starter-workflows/code-scanning/scorecards.yml#L28

**Comment by naveensrinivasan** [comment]
@laurentsimon

**Comment by laurentsimon** [comment]
good point. We're not using the starter-workflow in this repo. I created them to keep a copy from https://github.com/actions/starter-workflows/blob/main/code-scanning/scorecards.yml. I now realize that may be a pain to keep them in sync all the time. Shall we remove them from our repo? wdut?

**Comment by naveensrinivasan** [comment]
I think we should remove it.

**Comment by laurentsimon** [comment]
Can you send a PR for that? I'll LGTM. I'd like to keep track of how to do releases. There's an issue I created with steps to follow that indicate where to update the starter workflows. How about creating a RELEASE.md with those steps?

**Comment by naveensrinivasan** [comment]
We shouldn't remove this. Teams are directly using this action https://github.com/google/licensecheck/blob/db1987b6bdd30a6777a02509214b17063deff3c8/.github/workflows/scorecards-analysis.yml#L30

**Comment by laurentsimon** [comment]
what I mean is that the local file `scorecard-action/starter-workflows/code-scanning/scorecards.yml` is just a copy of the real file hosted at https://github.com/actions/starter-workflows/blob/main/code-scanning/scorecards.yml

The local copy can be removed without impacting any team.
